components/organisms/Layout.js

```diff
@@ -20,22 +22,56 @@ export const Layout = ({
   locale,
   langUrl,
 }) => {
+  //State for Search input
+  const [searchText, setSearchText] = useState(null);
+  //Handler for search input
+  function handleInputChange(e) {
+    setSearchText(e.target.value);
+  }
   const { t } = useTranslation("common");
   const language = locale === "en" ? "fr" : "en";
+
   return (
     <div className="overflow-x-hidden">
       <header>
         <PhaseBanner phase={t("Alpha")}>{t("alphaText")}</PhaseBanner>
-        <div className="layout-container">
-          <Link key={language} href={langUrl} locale={language}>
-            <a data-cy="toggle-language-link">
-              {language === "en" ? "English" : "Français"}
-            </a>
-          </Link>
-          <Header
-            headerLogoAltText="Symbol of the Government of Canada"
-            headerLogoImage="/sig-blk-en.svg"
-          />
+        <div className="layout-container ">
+          <div className="flex-col flex lg:flex lg:flex-row justify-between lg:mt-2 ">
```

Comment:
don't think this div is nessecary, try and reduce the amount of divs

Reply:
Div removed combined it with the other div for styling.